common/src/main/java/io/opentelemetry/android/common/internal/features/networkattrs/CurrentNetworkAttributesExtractor.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.android.internal.features.networkattrs;
+package io.opentelemetry.android.common.internal.features.networkattrs;
```

Comment:
I suppose networkattrs is shortening of networkattributes. 
Personally I suggest to use networkattributes so it is more obvious, attrs has already a meaning in tech. 
I undestand these were original changes so it is up to you :)